.github/workflows/ui_test.yml

```diff
@@ -20,7 +11,7 @@ jobs:
       - uses: actions/setup-node@v3
```

Comment:
These actions were throwing errors, since there are no tests being run here I've just turned them off until we have some.